binaries/cli/src/check.rs

```diff
@@ -16,19 +17,30 @@ pub fn check_environment() -> eyre::Result<()> {
 
     // check whether coordinator is running
     write!(stdout, "Dora Coordinator: ")?;
-    if coordinator_running()? {
-        let _ = stdout.set_color(ColorSpec::new().set_fg(Some(Color::Green)));
-        writeln!(stdout, "ok")?;
-    } else {
-        let _ = stdout.set_color(ColorSpec::new().set_fg(Some(Color::Red)));
-        writeln!(stdout, "not running")?;
-        error_occured = true;
-    }
+    let mut session = match connect_to_coordinator() {
+        Ok(session) => {
+            let _ = stdout.set_color(ColorSpec::new().set_fg(Some(Color::Green)));
+            writeln!(stdout, "ok")?;
+            Some(session)
+        }
+        Err(_) => {
+            let _ = stdout.set_color(ColorSpec::new().set_fg(Some(Color::Red)));
```

Comment:
I feel like we're re-implementing something in the likes of: https://docs.rs/color-eyre/latest/color_eyre/

Reply:
This is just for coloring the status of the coordinator and deamon in the output. For example: 
![image](https://user-images.githubusercontent.com/1131315/232773180-f6c88663-08dc-42a5-bf22-11605dbc5c4b.png)

So compared to `color_eyre`, it isnot directly related to error handling,

Reply:
Ah yeah okay. In that case, we can live it as is.